src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --primary-color: rgba(0, 172, 220, 1);
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif, system-ui;
+}
+
+.header {
+  /* border: 1px solid red; */
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header img {
+  display: block;
+}
+
+.nav-bar {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav-bar li {
+  /* border: 1px solid green; */
+  list-style: none;
+}
+
+.nav-list-elem a {
+  display: inline-block;
+  height: 60px;
+  line-height: 61px;
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  text-transform: uppercase;
+  font-weight: 500;
+}
```

Comment:
The class name 'nav-list-elem' does not clearly represent the meaning of the content. Consider using a more descriptive class name that indicates its role or function within the navigation bar.